app/defines/interrupt_handler.h

```diff
@@ -43,28 +43,24 @@ void interrupt_handler() {
         rvc_printf("TIMER_INTERRUPT\n");
```

Comment:
Instead of printing, we should call a new function called "machine_timer_routine ".
This routine should be accessible for the user SW developer so they can decide what they want to take place during a "machine_timer_interupt".

I will talk to ChatGPT and research the option of creating "default" functions that the user can override.

In this default function, we want a simple global int to increment with every interrupt.